### PR TITLE
Feature/hello

### PR DESCRIPTION
**Pull request by Kgomotso196**
# PROJECT TO DOs
### ON THIS PROJECT I:
1.  Added 3 files (each one in a separate commit):

    -     A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
    -     A .css file that includes one class with styles for your header (make it in your favorite color).
    -     A test.md file with the text "This file should be ignored by git"

2.  Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
3.  Made sure that I used correct ([GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)).
4.  Made sure that I documented my work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,181 +25,104 @@ After you're finished please remove all the comments and instructions!
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](https://kgomotso196.github.io/Hello-Microverse/)
```
Comment (Johnadibe) [inline README.md]: - [ ] Please kindly remove the Live Demo section since you are not deploying your project yet😊

File: README.md [hunk 2]
```diff
@@ -241,48 +131,22 @@ Feel free to check the [issues page](../../issues/).
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](https://github.com/Kgomotso196/Hello-Microverse/blob/new-project/MIT.md) licensed.
```
Comment (Johnadibe) [inline README.md]: - [ ] The link to your MIT license is broken. Please kindly fix that😊. This makes you to be the owner of your project.

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @Kgomotso196 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used GitHub Flow correctly 👍🏽
✅  Last file(test.md) is actually ignored by git✔️
✅  Node modules ignored by git✔️


## Required Changes ♻️
- [ ] Please kindly make sure that your linter results are all green. These analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs😊. 
As noticed in your project:

linters2

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @Kgomotso196, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**